RoaringBitmap/src/test/java/org/roaringbitmap/TestRoaringBitmap.java

```diff
@@ -2984,8 +2984,8 @@ public void testIterator() {
             rb.add(k * 100);
         }
         RoaringBitmap copy1 = new RoaringBitmap();
-        for (int x : rb) {
-            copy1.add(x);
+        for (Iterator<Integer> x = rb.iterator(); x.hasNext();) {
```

Comment:
We should demonstrate the usage without autoboxing, probably through .forEach(int -> XXX)